lib/dbadaptor.py

```diff
@@ -196,14 +197,14 @@ def create_table(self, conn, name, columns, keys, recreate):
                 for ky in auto_keys:
                     if ky in prim_keys and columns[ky] != 'INTEGER':
                         columns[ky] = 'INTEGER'
-        super(SQLDatabaseAdaptor, self).create_table(conn, name, columns, keys,
-                                                     recreate)
+        super(SQLDatabaseAdaptor, self).create_table(conn, name, columns, keys, recreate)
 
```

Comment:
I think this line break is due to the 79 charators limit. So I would keep it as it is.  Same for others...